account_financial_report/i18n/es.po

```diff
@@ -1353,6 +1360,11 @@ msgstr ""
 msgid "Partner Initial balance"
 msgstr "Saldo Inicial de empresa"
 
+#. module: account_financial_report
+#: model:ir.model.fields.selection,name:account_financial_report.selection__open_items_report_wizard__grouped_by__salesperson
+msgid "Partner Salesperson"
+msgstr "Comercial del socio"
```

Comment:
Never use the word `Socio` for translating `Partner`. "Socio" is used for example in membership context.

```suggestion
msgstr "Comercial de la empresa"
```